packages/core/admin/admin/src/components/GuidedTour/init.js

```diff
@@ -0,0 +1,21 @@
+import set from 'lodash/set';
+
+const init = initialState => {
+  const guidedTourLocaleStorage = JSON.parse(localStorage.getItem('GUIDED_TOUR_COMPLETED_STEPS'));
```

Comment:
Can you refer to my comment below in order to import the constants for the localStorage?